src/resolvers/Mutation/updatePost.ts

```diff
@@ -67,6 +69,19 @@ export const updatePost: MutationResolvers["updatePost"] = async (
     );
   }
 
+  // Check title and pinpost
+  if (args.data?.title && !post.pinned) {
+    throw new errors.InputValidationError(
```

Comment:
1. Pinned posts can only be created by Super Admins or Admins for the organization where the post was created.
2. Please make sure this logic is enforced